### PR TITLE
global: migrate to Flask-Security-Invenio and move UserIdentity

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: invenio_accounts/alembic/62efc52773d4_create_useridentity_table.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#
```
Comment (lnielsen) [inline invenio_accounts/alembic/62efc52773d4_create_useridentity_table.py:1]: Note: Another migration is to be added to Invenio-OAuthclient that moves the data from the previous table. Imports is kept backward compatible in invenio-oauthclient

File: invenio_accounts/ext.py [hunk 2]
```diff
@@ -16,41 +16,22 @@
...
 from . import config
 from .datastore import SessionAwareSQLAlchemyUserDatastore
-from .hash import InvenioAesEncryptedEmail, _to_binary
+from .hash import InvenioAesEncryptedEmail
 from .models import Role, User
 from .sessions import csrf_token_reset, login_listener, logout_listener
 from .utils import obj_or_import_string, set_session_info
 
 
-def get_hmac(password):
```
Comment (lnielsen) [inline invenio_accounts/ext.py:36]: No longer needed because it's fixed in Flask-Security-Invenio

File: invenio_accounts/ext.py [hunk 3]
```diff
@@ -69,28 +50,13 @@ def __init__(self, app=None, sessionstore=None):
...
-
         # Disable remember me cookie generation as it does not work with
         # session activity tracking (a remember me token will bypass revoking
-        # of  a session).
+        # of a session).
         def patch_do_nothing(*args, **kwargs):
             pass
         LoginManager._set_cookie = patch_do_nothing
 
-        # Disable loading user from headers and object because we want to be
```
Comment (lnielsen) [inline invenio_accounts/ext.py:87]: Disabled directly in flask-security-invenio

**Review by max-moser — APPROVED** [review]

File: setup.py [hunk 4]
```diff
@@ -106,6 +106,10 @@
             'invenio_accounts_role = invenio_accounts.admin:role_adminview',
             'invenio_accounts_session = '
             'invenio_accounts.admin:session_adminview',
+            'invenio_accounts_user_identity = '
+            'invenio_accounts.admin:user_identity_adminview',
+        ],
```
Comment (max-moser) [inline setup.py:111]: is this an extra `],`?

**Comment by lnielsen** [comment]
Superseded by #385

**Pull request closed without merging**